cranelift-codegen/src/verifier/mod.rs

```diff
@@ -1837,12 +1837,41 @@ impl<'a> Verifier<'a> {
         Ok(())
     }
 
+    fn verify_function_invalid_types(&self, errors: &mut VerifierErrors) -> VerifierStepResult<()> {
+        self.func
+            .signature
+            .params
+            .iter()
+            .filter(|&param| param.value_type == types::INVALID)
+            .for_each(|_| report!(errors, AnyEntity::Function, "Parameter has an invalid type"));
```

Comment:
It might be helpful to include the parameter that is invalid, e.g. `"Parameter {} has..."`

Reply:
Done. As the parameters have no names, I report the position of the parameter.